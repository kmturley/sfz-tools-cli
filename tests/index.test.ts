import { cli, CliOutput } from './shared';

test('Example', async () => {
  const output: CliOutput = await cli(`convert ./test/example.sfz --json`);
  expect(output.exitCode).toBe(0);
  expect(output.stdout).toMatchSnapshot();
});
