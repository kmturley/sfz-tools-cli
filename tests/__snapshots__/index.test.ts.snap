// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Get help 1`] = `
"Usage: index [options] [command]

Options:
  -V, --version                  output the version number
  -h, --help                     display help for command

Commands:
  analyze [options] <filepath>   Analyze audio files to detect audio features
  convert [options] <filepath>   Convert sfz files to other formats
  encode [options] <filepath>    Encode audio files into other formats
  organize [options] <filepath>  Organize files using pattern
  splice [options] <filepath>    Splice audio files into separate notes
  help [command]                 display help for command
"
`;
